IM_calculation/Advanced_IM/runlibs_2d.py

```diff
@@ -127,7 +130,6 @@ def main(args, im_name, run_script):
                 f"{component_outdir} failed to converge, skipping rest of the components"
             )
             model_converged = False
```

Comment:
Comment here and describe you are still computing even if one component fails - It'll be worth stating that potentially geom component can't be worked out.